.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: unconfirmed
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behaviour:
+1. happens with input (include input files)
+2. start `xtb` with (all the options here)
+3. output showing the error
```

Comment:
Please start  `xtb` with (all your options) and additionally "--verbose" and attach this ouput and all other outputs showing the error.

Reply:
sounds good.